index.html

```diff
@@ -1,66 +1,282 @@
+      Donec dui nulla, ultricies et convallis at, maximus non ante. Sed pellentesque vel arcu vel venenatis.
+      Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.
+      Morbi venenatis at odio vitae tincidunt. Etiam sollicitudin sit amet leo quis molestie. <span class="higlight-3">
+      Morbi eu mauris viverra, bibendum odio id, finibus erat. </span>  Sed rhoncus lacinia porttitor. Duis eleifend libero erat,
+      sit amet semper augue elementum quis. Proin leo sem, ultricies vitae augue pulvinar, dignissim tempus tellus.
+      Proin molestie nulla tortor, et auctor dolor pretium vitae. Donec posuere dui vel ante porttitor aliquet.
+      Ut dictum gravida turpis a lacinia. Pellentesque nunc lorem, condimentum a consequat at, luctus eu ante.
+      Cras nec dignissim lacus, at luctus odio. Nunc a lorem pharetra, vulputate libero auctor, venenatis dolor.
+      Donec dui nulla, ultricies et convallis at, maximus non ante. Sed pellentesque vel arcu vel venenatis.
+      Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.
+      Morbi venenatis at odio vitae tincidunt. Etiam sollicitudin sit amet leo quis molestie. <span class="higlight-3">
+      Morbi eu mauris viverra, bibendum odio id, finibus erat. </span>  Sed rhoncus lacinia porttitor. Duis eleifend libero erat,
+      sit amet semper augue elementum quis. Proin leo sem, ultricies vitae augue pulvinar, dignissim tempus tellus.
+      Proin molestie nulla tortor, et auctor dolor pretium vitae. Donec posuere dui vel ante porttitor aliquet.
+      Ut dictum gravida turpis a lacinia. Pellentesque nunc lorem, condimentum a consequat at, luctus eu ante.
+      Cras nec dignissim lacus, at luctus odio. Nunc a lorem pharetra, vulputate libero auctor, venenatis dolor.
+      Donec dui nulla, ultricies et convallis at, maximus non ante. Sed pellentesque vel arcu vel venenatis.
+      Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.
+      Morbi venenatis at odio vitae tincidunt. Etiam sollicitudin sit amet leo quis molestie. <span class="higlight-3">
+      Morbi eu mauris viverra, bibendum odio id, finibus erat. </span>  Sed rhoncus lacinia porttitor. Duis eleifend libero erat,
+      sit amet semper augue elementum quis. Proin leo sem, ultricies vitae augue pulvinar, dignissim tempus tellus.
+      Proin molestie nulla tortor, et auctor dolor pretium vitae. Donec posuere dui vel ante porttitor aliquet.
+      Ut dictum gravida turpis a lacinia. Pellentesque nunc lorem, condimentum a consequat at, luctus eu ante.
+      Cras nec dignissim lacus, at luctus odio. Nunc a lorem pharetra, vulputate libero auctor, venenatis dolor.
+      Donec dui nulla, ultricies et convallis at, maximus non ante. Sed pellentesque vel arcu vel venenatis.
+      Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.
+    </p>
+
+    <div class="flex-box">
```

Comment:
Veszélyes dolog classnevekbe beleírni a class tartalmát. Ha olvasom, akkor azt fogom feltételezni, hogy ez csak egy `display: flex`-et ad ehhez a divhez, viszont a style.css-ben ennél több van. Ha pedig változik a display érték, mondjuk `grid`-re, akkor hazudni fog a classnév -> kénytelen leszel megváltoztatni a nevét is a stílusával együtt. És minden egyes helyen, ahhol használtad.